ios/AccountsSDK.swift

```diff
@@ -3,178 +3,176 @@ import TicketmasterTickets
-      resolve(data)
-    } aborted: { oldAuthToken, backend in
-      print("Token Retrieval Aborted ")
-      let data = ["accessToken": ""]
-      resolve(data)
-    } failure: { oldAuthToken, error, backend in
-      print("Token Retrieval Error: \(error.localizedDescription)")
-      reject( "Accounts SDK Token Retrieval Error", error.localizedDescription, error as NSError)
-    }
-  }
-  
-  @objc public func isLoggedIn(_ resolve: @escaping ([String: Bool]) -> Void, reject: @escaping (_ code: String, _ message: String, _ error: NSError) -> Void) {
     
-    TMAuthentication.shared.memberInfo { memberInfo in
-      guard let id = memberInfo.globalID else {
-        resolve(["result": false])
-        return
-      }
-      
-      let hasToken = TMAuthentication.shared.hasToken()
-      resolve(["result": hasToken])
-      
-    } failure: { oldMemberInfo, error, backend in
-      if(TMAuthentication.shared.hasToken()){
-        let hasToken = TMAuthentication.shared.hasToken()
-        resolve(["result": hasToken])
-      } else {
-        reject("Accounts SDK Is Logged In Error", error.localizedDescription, error as NSError)
-      }
+    
+    func sendEvent(_ name: String, body: [String : Any]) {
+        EventEmitter.emitter.sendEvent(withName: name, body: body)
     }
-  }
-  
-  func onStateChanged(backend: TicketmasterAuthentication.TMAuthentication.BackendService?, state: TicketmasterAuthentication.TMAuthentication.ServiceState, error: (Error)?) {
-    print("Backend TicketmasterAuthentication \(state.rawValue)")
-    switch state{
-    case .serviceConfigurationStarted:
-      return
-    case .serviceConfigured:
-      return
-    case .serviceConfigurationCompleted:
-      return
-    case .loginStarted:
-      // Replace return with below to send Native event to React Native side
-      // self.sendEvent(withName: "loginStarted", body: ["loginStarted:": "loginStarted event sent"])
-      return
-    case .loginPresented:
-      return
-    case .loggedIn:
-      return
-    case .loginAborted:
-      return
-    case .loginFailed:
-      return
-    case .loginLinkAccountPresented:
-      return
-    case .loginCompleted:
-      return
-    case .tokenRefreshed:
-      return
-    case .logoutStarted:
-      return
-    case .loggedOut:
-      return
-    case .logoutCompleted:
-      return
-    @unknown default:
-      return
+    
+    func onStateChanged(backend: TicketmasterAuthentication.TMAuthentication.BackendService?, state: TicketmasterAuthentication.TMAuthentication.ServiceState, error: (Error)?) {
+        print("Backend TicketmasterAuthentication \(state.rawValue)")
+        switch state{
+        case .serviceConfigurationStarted:
+            return
+        case .serviceConfigured:
+            return
+        case .serviceConfigurationCompleted:
+            return
+        case .loginStarted:
+            sendEvent("igniteAnalytics", body: ["AccountsSDK - loginStarted:": "loginStarted event sent"])
```

Comment:
key should probably just be camelCased just to make easier to access on the RN side, and avoid any lint issues. `accountsSDKLoginStarted` or something

Reply:
True, dummy format for now but will cause issues if it stayed like that, will update this 👍